conda/plugins/types.py

```diff
@@ -488,3 +492,22 @@ class CondaEnvironmentSpecifier(CondaPlugin):
 
     name: str
     environment_spec: type[EnvironmentSpecBase]
+
+
+@dataclass
+class CondaEnvironmentExporter(CondaPlugin):
+    """
+    **EXPERIMENTAL**
+
+    Return type to use when defining a conda environment exporter plugin hook.
+
+    :param name: name of the exporter (e.g., ``environment-yaml``)
+    :param aliases: user-friendly format aliases (e.g., ("yaml",))
+    :param default_filenames: default filenames this exporter handles (e.g., ("environment.yml", "environment.yaml"))
+    :param export: callable that exports an Environment to string format
+    """
+
+    name: str
+    aliases: tuple[str, ...]
```

Comment:
nice one! 
Do these aliases also need to comply with the name rules and name collision mechanisms introduced in https://github.com/conda/conda/pull/15002?